tests/Unit/Pypp/CheckWithRandomValues.hpp

```diff
@@ -143,7 +143,7 @@ void check_with_random_values_impl(
     std::index_sequence<ResultIs...> /*index_return_types*/,
     std::index_sequence<ArgumentIs...> /*index_argument_types*/,
     std::index_sequence<MemberArgsIs...> /*index_member_args*/,
-    TagsList /*meta*/, const double epsilon = 1.0e-12) {
+    TagsList /*meta*/, double epsilon = 1.0e-12) {
```

Comment:
this is fixed more correctly in #1389 so my vote is let's ignore it :) clang-tidy is wrong here

Reply:
Alright, sounds good. Removed.